src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/BulkAddSinglesControllerTest.java

```diff
@@ -59,7 +59,8 @@ public void shouldReturnValidationResponse() {
         var errors = List.of("Error 1", "Error 2", "Error 3");
         when(bulkAddSinglesValidator.validate(multipleDetails, authToken)).thenReturn(errors);
 
-        var response = bulkAddSinglesController.bulkAddSingleCasesImportFileMidEventValidation(multipleRequest, authToken);
+        var response = bulkAddSinglesController
```

Comment:
var

Reply:
As already discussed I changed 88 files which have around 3000 vars, may be we need a separate ticket for that

Reply:
removed all vars from this file